packages/fxa-auth-server/test/local/error.js

```diff
@@ -8,13 +8,14 @@ const { assert } = require('chai');
 const verror = require('verror');
 // This import is bad now!
 const messages = require('joi/lib/messages');
+console.log("MESSAGES", messages);
 const AppError = require('../../lib/error');
 const OauthError = require('../../lib/oauth/error');
 
 describe('AppErrors', () => {
   it('tightly-coupled joi message hack is okay', () => {
-    assert.equal(typeof messages.errors.any.required, 'string');
-    assert.notEqual(messages.errors.any.required, '');
+    assert.equal(typeof messages, 'string');
```

Comment:
nvm this file messages used to be https://npmtest.github.io/node-npmtest-joi/build/coverage.html/node-npmtest-joi/node_modules/joi/lib/language.js.html in this format now i think u need types/any/messages but dont know how to properly import